### PR TITLE
update bicbucstriim to v1.3.6 (previous v1.2.5)

### PR DESCRIPTION
**Pull request by alexandregz**

### CODE REVIEW

File: spk/bicbucstriim/Makefile [hunk 1]
```diff
@@ -1,18 +1,18 @@
 SPK_NAME = bicbucstriim
-SPK_VERS = 1.2.5
-SPK_REV = 4
+SPK_VERS = 1.3.6
+SPK_REV = 1
```
Comment (Dr-Bean) [inline spk/bicbucstriim/Makefile]: SPK_REV doesn't reset. It only goes up.

File: spk/bicbucstriim/Makefile [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
 SPK_ICON = src/bicbucstriim.png
 DSM_UI_DIR = app
 
 DEPENDS  = cross/$(SPK_NAME)
 
-MAINTAINER = Diaoul
+MAINTAINER = Alexandre Espinosa Menor
```
Comment (Dr-Bean) [inline spk/bicbucstriim/Makefile]: MAINTAINER is the original developer of the package, or alternatively, @Diaoul and you have agreed that you'll take over the maintenance of the package in the future.

**Comment by Dr-Bean** [comment]
Thanks for contributing. I have added two comments on the code. 
Have you verified that upgrading from the current 1.2.x to this package works as expected?

**Comment by alexandregz** [comment]
@Dr-Bean Yep, verified on my NAS, working with this package

**Comment by Dr-Bean** [comment]
Published bicbucstriim 1.3.6-5 on the repository. Thanks!

**Pull request merged**